index.js

```diff
@@ -298,22 +298,39 @@ WBuf.prototype.writeUInt32BE = function writeUInt32BE(v) {
   }
 };
 
-WBuf.prototype.writeUInt16LE = function writeUInt16LE(num) {
-  var r = ((num & 0xff) << 8) | (num >>> 8);
+WBuf.prototype.writeUInt16LE = function writeUInt16LE(v) {
+  var r = ((v & 0xff) << 8) | (v >>> 8);
   this.writeUInt16BE(r);
 };
 
-WBuf.prototype.writeUInt24LE = function writeUInt24LE(num) {
-  var r = ((num & 0xff) << 16) | (((num >>> 8) & 0xff) << 8) | (num >>> 16);
+WBuf.prototype.writeUInt24LE = function writeUInt24LE(v) {
+  var r = ((v & 0xff) << 16) | (((v >>> 8) & 0xff) << 8) | (v >>> 16);
   this.writeUInt24BE(r);
 };
 
-WBuf.prototype.writeUInt32LE = function writeUInt32LE(num) {
-  var r = ((num & 0xff) << 24) |
-          (((num >>> 8) & 0xff) << 16) |
-          (((num >>> 16) & 0xff) << 8) |
-          (num >>> 24);
-  this.writeUInt32BE(r);
+WBuf.prototype.writeUInt32LE = function writeUInt32LE(v) {
```

Comment:
What about this one?

Reply:
This does not work as it is currently implemented. It will overflow and result in an error in case I revert that change.